src/loader.c

```diff
@@ -236,6 +241,11 @@ bool parse_database(void *base, const char *name, const char *connstr)
 			res_pool_size = atoi(val);
 		} else if (strcmp("max_db_connections", key) == 0) {
 			max_db_connections = atoi(val);
+		} else if (strcmp("host_strategy", key) == 0) {
```

Comment:
This needs an entry in `config.md` to describe its usage.

Reply:
Documentation is in. Feedback welcome.